eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -68,20 +63,6 @@ func (c *ClusterLoader) Apply() (err error) {
 			Ready:     true,
 		},
 	}
-
-	// Wait for job to complete -- 2 hours because larger tests take a very long time.
-	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Hour)
-	defer cancel()
-	job := &v1.Job{}
-	for job.Status.Succeeded < 1 {
```

Comment:
Can you explain why we are getting rid of this? It just waits for at least one job is successful...

Reply:
We were going to use it for when we put certain tests into Prow, but found that this didn't properly handle the timeouts. Rather than trying to figure out the timeouts, I decided to get rid of it as we didn't need it, and may not need it later.

Reply:
> didn't properly handle the timeouts

Then, we can decrease the timeouts and just log in warning level? Do you run clusterloader2 for more than 2 hours?

Reply:
Depending on the size of the test, yes it can run for up to two hours. But when scale tests get too large, API calls time out, and the testing for that takes a very long time, so rather than having a partially working feature, I just wanted to remove it.

Reply:
> Depending on the size of the test, yes it can run for up to two hours. But when scale tests get too large, API calls time out, and the testing for that takes a very long time, so rather than having a partially working feature, I just wanted to remove it.

I understand that. Still, this breaks other small-scale tests where 2-hour is enough. Our bug bash job relies on this checks to make sure 5 rounds of clusterloader runs complete successfully.

We can add another field to `eksconfig` to opt out of this.

Reply:
Actually, let's skip this (delete). I was confused with other wati func.